### PR TITLE
add flash drives to special equipment task

### PR DESCRIPTION
**Pull request by Razzmatazzz**

### CODE REVIEW

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

File: src/tarkov-data-manager/index.js [hunk 1]
```diff
@@ -566,7 +558,7 @@ app.get('/items', async (req, res) => {
                                 <input id="source-upload" type="file" name="source-upload" />
                             </div>
                         </div>
-                        <div class="row>
+                        <div class="row">
```
Comment (GrantBirki) [inline src/tarkov-data-manager/index.js:561]: yoinks

File: src/tarkov-data-manager/jobs/update-quests.js [hunk 2]
```diff
@@ -932,8 +932,10 @@ module.exports = async (externalLogger = false) => {
             if (changedQuests[questData.id] && changedQuests[questData.id].objectivesAdded) {
                 for (const newObj of changedQuests[questData.id].objectivesAdded) {
                     if (questData.objectives.some(obj => obj.id === newObj.id)) {
+                        console.log(`${questData.name} skipping objective`,newObj)
```
Comment (GrantBirki) [inline src/tarkov-data-manager/jobs/update-quests.js:935]: Not sure if these are meant to be left in here but I think its fine either way 🤷‍♂️

**Review by GrantBirki — DISMISSED** [review]

File: src/tarkov-data-manager/index.js (hunk 1 above)
Comment (Razzmatazzz) [reply src/tarkov-data-manager/index.js:561]: yeah that was a whoopsie

File: src/tarkov-data-manager/jobs/update-quests.js (hunk 2 above)
Comment (Razzmatazzz) [reply src/tarkov-data-manager/jobs/update-quests.js]: Nope, note needed to keep

**Comment by Razzmatazzz** [comment]
.deploy

**Review by Shebuka — DISMISSED** [review]

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Review by Shebuka — APPROVED** [review]

**Pull request merged**